src/pages/Landing.js

```diff
@@ -2,31 +2,48 @@ import React from 'react';
 import { hot } from 'react-hot-loader/root';
 import Footer from '@makerdao/ui-components-footer';
 import Header from '@makerdao/ui-components-header';
-import { Box } from '@makerdao/ui-components-core';
-
+import { Box, Flex, Card, Text, Link } from '@makerdao/ui-components-core';
 import LandingHeroLayout from 'layouts/LandingHeroLayout';
+import { Routes } from '../utils/constants';
 
 import { Title } from 'components/Typography';
 
+const CardLink = ({ title, link }) => (
+  <Box mx="l">
+    <Link href={`/${link}`}>
+      <Card width="400px" height="250px">
+        <Flex
+          flexDirection="column"
+          justifyContent="center"
+          alignItems="center"
+          height="100%"
+        >
+          <Text.h2>{title}</Text.h2>
+          <br />
+
+          {/* Delete below later */}
+          <Text.h1>&#10230;</Text.h1>
+        </Flex>
+      </Card>
+    </Link>
+  </Box>
+);
+
 function Landing() {
   return (
     <Box width="100%">
       <Header />
       <Box bg="backgroundGrey">
         <LandingHeroLayout>
-          <Box
-            pb="m"
-            css={`
-              max-width: 500px;
-            `}
-          >
-            <Box pb="s">
-              <Title display="block">Oasis</Title>
-              <p>
-                CDP portal has been moved to <a href="/borrow">/borrow</a>
-              </p>
-            </Box>
-          </Box>
+          <Flex flexDirection="column">
+            <Title pb="l" textAlign="center">
+              Oasis
+            </Title>
+            <Flex pb="l" flexDirection="row" justifyContent="space-around">
+              <CardLink title="Borrow" link={Routes.BORROW} />
+              <CardLink title="Save" link={Routes.SAVE} />
+            </Flex>
+          </Flex>
         </LandingHeroLayout>
```

Comment:
Rough outline for the Oasis Landing page. Just provides two Card links for the relevant sections